build/patches/Enable-crash-reporting-ui.patch

```diff
@@ -111,6 +331,46 @@ diff --git a/components/crash_strings.grdp b/components/crash_strings.grdp
 +    Save now
    </message>
  </grit-part>
+diff --git a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java
+--- a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java
++++ b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java
+@@ -107,6 +107,8 @@ public class CrashFileManager {
+ 
+     private static final Pattern TMP_PATTERN = Pattern.compile("\\.tmp\\z");
+ 
++    private static final String UPLOADED_MINIDUMP_ZIP_SUFFIX = ".zip";
```

Comment:
There is not really any upload, shouldn't we call it `SAVED_MINIDUMP_ZIP_SUFFIX`?

Reply:
Yes you are right.